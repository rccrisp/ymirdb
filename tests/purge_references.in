set a 1 2 3
set b a
purge a
snapshot
set b 4 5 6
snapshot
purge a
drop 1
purge a
list entries
bye
