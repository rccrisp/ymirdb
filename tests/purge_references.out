> ok

> ok

> not permitted

> saved as snapshot 1

> ok

> saved as snapshot 2

> not permitted

> ok

> ok

> b [4 5 6]

> bye
